package-lock.json

```diff
@@ -1,10910 +1,13 @@
 {
   "name": "@triniti/cms",
-  "version": "0.13.3",
-  "lockfileVersion": 2,
+  "version": "0.13.4",
+  "lockfileVersion": 1,
```

Comment:
@rsumilang did you intend to revert the lock file version?

Reply:
@efjacobson no, `npm up` did that automatically. if i change it back to `2` and run `npm install` again then it changes it back to `1`.

Reply:
@rsumilang what version of npm are you using?

Reply:
v10.13.0

Reply:
@rsumilang not node, npm. if you are using npm 7 it will use lockfileVersion 2. for ex i am using npm 7.7.0

Reply:
This is what I get with the `nvm use` command: `Now using node v10.13.0 (npm v6.4.1)`

Reply:
@rsumilang yeah you are using npm 6.4.1 - if you update npm itself to 7 it will use the new lockfileVersion